icu4c/source/i18n/numrange_impl.cpp

```diff
@@ -128,7 +129,16 @@ NumberRangeFormatterImpl::NumberRangeFormatterImpl(const RangeMacroProps& macros
     getNumberRangeData(macros.locale.getName(), nsName, data, status);
     if (U_FAILURE(status)) { return; }
     fRangeFormatter = data.rangePattern;
-    fApproximatelyModifier = {data.approximatelyPattern, kUndefinedField, false};
+
```

Comment:
I should stop loading `data.approximatelyPattern` to save some time in the resource bundle.